src/main/scala/io/iohk/ethereum/crypto/ECDSASignature.scala

```diff
@@ -72,6 +74,19 @@ object ECDSASignature {
     else s
   }
 
+  def sign(message: Array[Byte], keyPair: AsymmetricCipherKeyPair): ECDSASignature = {
+    val signer = new ECDSASigner(new HMacDSAKCalculator(new SHA256Digest))
+    signer.init(true, keyPair.getPrivate)
+    val components = signer.generateSignature(message)
+    val r = components(0)
+    val s = ECDSASignature.canonicalise(components(1))
+    val v = ECDSASignature
+      .calculateV(r, s, keyPair, message)
+      .getOrElse(throw new RuntimeException("Failed to calculate signature rec id"))
```

Comment:
We use that for signing and I don't think the result should be dependent on user input (correct me if I'm wrong) - if there's an error it means we messed up. An exception is fine is such situation